Header header

std_msgs/String[] labels
cloud_get_object/Objects_Pixels_Vector objects_vector
recognize_with_vgg/Object_Pixels_Boundaries[] boundaries
geometry_msgs/Quaternion support_plane
std_msgs/UInt16MultiArray[] hsv_vector